### ISSUE TITLE
Cannot run the code

### ISSUE THREAD

**Issue by guangnianchenai** [DeCapsNet/DeCapsNet-demo#1]
Hi, it is a great work for depression detection and thanks for sharing your code. But I meet some problems when running the code. The biggest one is processing datasets. Could you tell me that how can I obtain the three datasets and use them to training the proposed model? I could not find the link to download them. Thank you for your help.

**Comment by ShengwuZheng** [comment]
> Hi, it is a great work for depression detection and thanks for sharing your code. But I meet some problems when running the code. The biggest one is processing datasets. Could you tell me that how can I obtain the three datasets and use them to training the proposed model? I could not find the link to download them. Thank you for your help.

RSDD and TRT are hard to access now. But eRisk can be obtained by sending a application form to the organizers. Access the eRisk official web page for more information

**Comment by ShengwuZheng** [comment]
Hi. Thanks for your work. I find in your code that loss = 【100】 * margin_loss + alpha * l1 + beta * cl1 + gamma * cl2. Why does the margin loss multiply 100 ?
